packages/fx-core/src/plugins/resource/bot/resources/messages.ts

```diff
@@ -46,6 +46,8 @@ export class Messages {
     "Falling back to use local template zip.";
   public static readonly ResourceProviderExist = (rp: string): string =>
     `Resource Provider ${rp} already exists, do not need to register.`;
+  public static readonly BotResourceExist = (where: string): string =>
+    `Bot resource already existed on ${where}, skip creating Bot resource.`;
```

Comment:
Nit: The bot resource is already existing.

Reply:
I think Yitong's wording is OK, no need to change